test/integration_test.exs

```diff
@@ -0,0 +1,138 @@
+defmodule Membrane.RTC.Engine.TimescaleDB.IntegrationTest do
+  use ExUnit.Case
+
+  alias Membrane.RTC.Engine.TimescaleDB.{
+    Cleaner,
+    Model,
+    Repo
+  }
+
+  alias Model.{
+    PeerMetrics,
+    TrackMetrics
+  }
+
+  setup do
+    Ecto.Adapters.SQL.Sandbox.checkout(Repo)
+    :ok
+  end
+
+  describe "Reporter stores" do
+    test "report in the db" do
+      Model.insert_report(Repo, rtc_engine_report_1())
+
+      assert Repo.all(PeerMetrics) |> length() == 2
+      assert Repo.all(TrackMetrics) |> length() == 3
+
+      Ecto.Adapters.SQL.Sandbox.checkin(Repo)
+    end
+
+    test "multiple reports in the db" do
+      Model.insert_report(Repo, rtc_engine_report_1())
+      Model.insert_report(Repo, rtc_engine_report_2())
+
+      assert Repo.all(PeerMetrics) |> length() == 3
+      assert Repo.all(TrackMetrics) |> length() == 4
+
+      Ecto.Adapters.SQL.Sandbox.checkin(Repo)
```

Comment:
It seems not to be required to check in the connection manually, the example doesn't: https://hexdocs.pm/ecto_sql/Ecto.Adapters.SQL.Sandbox.html#module-example